### DISCUSSION TITLE
Search bar mode game, country, SEARCH BY: (IP, SERVER CURRENT MAP, ETC)

### DISCUSSION THREAD

**Discussion by Kent-Romania** [tltneon/lgsl#131] [General]
**Hello, is it possible to implement in lgsl a search bar servers and modes according to criteria as in attach?
it helps me a lot. thanks**
![s1](https://user-images.githubusercontent.com/93764409/187553192-883df590-9486-48dd-929f-7788ccf74a88.png)

**Comment by BaxAndrei** [comment]
Cred ca deja s-a adaugat, dar in noua versiune in lucru https://github.com/tltneon/lgsl/commit/9111f5c4a7c7e389deedafd07d6f3a956f6b9f9f
